test/Elastic.Apm.SqlClient.Tests/SqlClientListenerTests.cs

```diff
@@ -41,10 +41,12 @@ public static IEnumerable<object[]> Connections
 			get
 			{
 				yield return new object[] { new Func<string, DbConnection>(connectionString => new SqlConnection(connectionString)) };
+#if !NETFRAMEWORK
```

Comment:
If we update `"Microsoft.Data.SqlClient"` to `Version="1.1.1"` I think this'll also work on Full Framework. I think there was a bug in the older version that made it not working - I was able to repro the problem that [this PR](
https://github.com/dotnet/SqlClient/pull/241) fixes, after that this worked for me on FullFramework with the `SqlEventListener`. 

Once this works, some of the `#if`s can be also updated.

Reply:
Nice catch. I'll update it